pipeline/batch.go

```diff
@@ -53,7 +68,8 @@ type Batcher struct {
 	maintenanceFn       BatcherMaintenanceFn
 	controller          OutputPluginController
 	workerCount         int
-	batchSize           int
+	batchQuantity       int
+	batchSizeBytes      int
```

Comment:
Let's use same naming everywhere. `maxSizeBytes/maxSizeCount`, `batchSizeBytes/batchSizeCount` for example